app/assets/javascripts/application/tools/congress_message.js

```diff
@@ -4,6 +4,43 @@ $(document).on("ready", function() {
     $("input[type=submit]").hide();
     $("input,textarea,button,select", $(this)).attr("disabled", "disabled");
   });
+  $("#to-page-2").click(function(){  // show customize message
+    $("#customize-message").show();
+    $(".address-lookup").hide();
+    $("#to-page-2").hide();
+    $(".page-indicator div.page1").css({
+      'background-color': '#2D2D2D',
+      'color': '#BABABA'
+    });
+    $(".page-indicator div.page2").css({
+      'background-color': '#BABABA',
+      'color': '#2D2D2D'
+    });
+  });
+
+  $("#to-page-3").click(function(){ // show representatives info
+    $("#customize-message").hide();
+    $("#to-page-3").hide();
+    $(".page-indicator div.page2").css({
+      'background-color': '#2D2D2D',
+      'color': '#BABABA'
+    });
+    $(".page-indicator div.page3").css({
+      'background-color': '#BABABA',
+      'color': '#2D2D2D'
+    });
+  });
+
+  $("#tools #congress-message-tool input[type='submit'].test").click(function(){ // show thank you
+
+  });
```

Comment:
oh, sorry, one last thing! this click handler can be deleted now

Reply:
Oh, right!